connectors/sources/confluence.py

```diff
@@ -262,6 +262,7 @@ def get_default_configuration(cls):
             "spaces": {
                 "display": "textarea",
                 "label": "Confluence space keys",
+                "tooltip": "This configurable field is ignored when Advanced Sync Rules are used.",
```

Comment:
Could you keep these alphabetised? `tooltip` should go after `order` here.

Reply:
Updated with commit id: [803c757](https://github.com/elastic/connectors-python/pull/1025/commits/803c7574c4f2fd2ddce1c94e9867accbbb87edd2)